src/PropertyList/PropertyList.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactNode } from 'react';
+import Immutable from 'immutable';
+import { AttributeGrid, AttributeList, Caption, Content } from './propertlyListStyles';
```

Comment:
minor: Mispelled file name. Filename should be `propertyListStyles.tsx` (and all imports referencing should be updated).